src/__swaps__/screens/Swap/components/ReviewPanel.tsx

```diff
@@ -370,10 +371,23 @@ export function ReviewPanel() {
                   <View style={sx.gasContainer}>
                     <AnimatedChainImage showMainnetBadge asset={internalSelectedInputAsset} size={16} />
                   </View>
-                  <Inline horizontalSpace="4px">
-                    <EstimatedGasFee />
-                    <EstimatedArrivalTime />
-                  </Inline>
+                  <Stall
+                    isStalledWorklet={() => {
+                      'worklet';
+                      return configProgress.value !== NavigationSteps.SHOW_REVIEW;
+                    }}
+                    placeholder={
+                      <Inline horizontalSpace="4px">
+                        <EstimatedSwapGasFeeSlot text="--" align="left" color="label" size="15pt" weight="heavy" />
+                        {null}
```

Comment:
was this `{null}` on purpose?

Reply:
yeah it was to show that it's not rendering any placeholder for `<EstimatedArrivalTime />`, but keeping the same tree structure with `<Inline />` so it's easier on react